runtime/src/runtime/fvm.rs

```diff
@@ -266,6 +266,38 @@ where
         })
     }
 
+    fn user_get_beacon_randomness(
```

Comment:
We don't need to duplicate code. Let's just:

1. Provide default `get_randomness_from_*` implementations that call the `user_` functions.
2. Implement the user ones everywhere, not the current ones.

Also, let's stick to the current naming conventions. `user_get_randomness_from_beacon` or `get_user_randomness_from_beacon` or `get_randomness_from_beacon_user`, etc.